pkg/router/controller/host_admitter.go

```diff
@@ -227,8 +231,33 @@ func (p *HostAdmitter) displacedRoutes(newRoute *routeapi.Route) ([]*routeapi.Ro
 	// See if any existing wildcard routes block our domain, or if we displace them
 	for i, route := range p.claimedWildcards[wildcardKey] {
 		if route.Namespace == newRoute.Namespace {
-			// Never displace a route in our namespace
```

Comment:
Its a double check but I added it in. Yeah, eventually need to merge the two. This ballooned into more.